4.3-debug/.config

```diff
@@ -193,7 +196,7 @@ CONFIG_COMPAT_BRK=y
 CONFIG_SLUB=y
 # CONFIG_SLOB is not set
 CONFIG_SLUB_CPU_PARTIAL=y
-CONFIG_SYSTEM_TRUSTED_KEYRING=y
+# CONFIG_SYSTEM_DATA_VERIFICATION is not set
```

Comment:
It was not disabled, it was just moved here https://github.com/scaleway/kernel-tools/pull/183/files#diff-cff12f72534ea17ed126530ef88c1ed9R3864